src/Courses/TopicSettings/GenericFormInputs.tsx

```diff
@@ -221,7 +221,7 @@ export const MaxVersionsField: React.FC<{}> = () => {
             name={`${examFieldNamePrefix}.maxVersions`}
             InputLabelProps={{ shrink: true }}
             inputRef={register()}
-            label={'Number of Versions'}
+            label={'Max Versions'}
```

Comment:
I don't understand why this is changed from 'Number of Versions' to 'Max Versions' -- "max" is definitely shorthand